backend/apps/monitoring/celery_monitor.py

```
@@ -0,0 +1,265 @@
+                "started_at": timezone.now(),
+                "args_summary": args_repr,
+            },
+        )
+        if not created:
+            task_run.status = "STARTED"
+            task_run.started_at = timezone.now()
+            task_run.args_summary = args_repr
+            task_run.save()
+
+        broadcast_task_update(task_run)
+    except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)